svc/svc.go

```diff
@@ -7,6 +7,10 @@
 //		env := svc.InitAll()
 //		defer env.Close()
 //
+//		// Report panics in main()
+//		ctx := reporter.WithReporter(context.Background(), env.Reporter)
```

Comment:
any reason this can't be in `svc.InitAll()`?

Reply:
Well, I could add that to InitAll() and return the context in the env struct, so it would look like:

``` golang
env := svc.InitAll()
defer reporter.Monitor(env.Context())
defer env.Close()
```

That's better I think, yeah? The `Monitor()` call has to be in main() though

Reply:
Actually I think I can include the Monitor in the Close() func

Reply:
yea, that's what I was thinking

```
func InitAll() Env {	
  	ctx := reporter.WithReporter(context.Background(), env.Reporter)
	...
  	return Env{
		Logger:   l,
		Reporter: InitReporter(),
		Close: func() {
			traceCloser()
			metricsCloser()
			reporter.Monitor(ctx)
		},
	}